### PR TITLE
Add the Bug Fables wiki

### LINKED ISSUE #631: Add a wiki: Bug Fables Wiki
Link to origin wiki: https://bugfables.fandom.com/wiki/Bug_Fables_Wiki
Link to destination (independent) wiki: https://bugfables.miraheze.org/wiki/Bug_Fables_Wiki

The bug fables community is migrating to miraheze. I have volunteered myself the past weeks to lead the migration to a satisfactory state. It is now ready to be promoted and as a first step, I would like to request the new wiki to be registered on indie wiki buddy.

### PR DESCRIPTION
**Pull request by aldelaro5**
Closes #631 

Figured I could pr this after submitting the issue. Let me know if anything seems out of place.

### CODE REVIEW

File: data/sitesEN.json [hunk 1]
```diff
@@ -6424,5 +6424,24 @@
     "destination_main_page": "Main_Page",
     "destination_search_path": "/w/index.php",
     "destination_content_path": "/wiki/"
+  },
+  {
+    "id": "en-bug-fables",
```
Comment (KevinPayravi) [inline data/sitesEN.json:6429]: Please make the ID `en-bugfables` (we only use the hyphen once to separate the language)

File: data/sitesEN.json [hunk 2]
```diff
@@ -6424,5 +6424,24 @@
...
+        "origin_main_page": "Bug_Fables_Wiki"
+      }
+    ],
+    "destination": "Bug Fables Wiki",
+    "destination_base_url": "bugfables.miraheze.org",
+    "destination_platform": "mediawiki",
+    "destination_icon": "bugfables.png",
+    "destination_main_page": "Bug_Fables_Wiki",
+    "destination_search_path": "/w/index.php",
+    "destination_content_path": "/wiki/"
```
Comment (KevinPayravi) [inline data/sitesEN.json:6445]: Please add a `miraheze` tag (see Adventure Zone Wiki for an example: https://github.com/KevinPayravi/indie-wiki-buddy/blob/main/data/sitesEN.json#L86).

```
"tags": [
  "miraheze"
]
```

**Review by KevinPayravi — CHANGES_REQUESTED** [review]
Thank you! Requesting two small changes

File: data/sitesEN.json (hunk 2 above)
Comment (aldelaro5) [reply data/sitesEN.json]: done

File: data/sitesEN.json (hunk 1 above)
Comment (aldelaro5) [reply data/sitesEN.json]: done

**Review by KevinPayravi — APPROVED** [review]

**Comment by KevinPayravi** [comment]
Thank you!

**Pull request merged**